HW18andmore/01.txt

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE mynames(
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    first_name TEXT,
+    last_name TEXT,
+    age REAL
```

Comment:
I don't think `REAL` is the correct type for a person's age, use INTEGER instead of it

Reply:
Please see new commit with INTEGER for person's age